tests/model/test_labels.py

```
@@ -587,3 +592,38 @@ def test_make_training_splits_save(slp_real_data, tmp_path):
     assert train_.provenance["source_labels"] == slp_real_data
     assert val_.provenance["source_labels"] == slp_real_data
     assert test_.provenance["source_labels"] == slp_real_data
+
+
+@pytest.mark.parametrize("embed", [True, False])
+def test_make_training_splits_save(slp_real_data, tmp_path, embed):
```

Ruff

Redefinition of unused `test_make_training_splits_save` from line 577

(F811)